ops/services/okta-app/main.tf

```diff
@@ -68,3 +69,17 @@ resource "okta_auth_server_scope" "sr_env" {
   metadata_publish = "NO_CLIENTS"
   default          = false
 }
+
+resource "okta_trusted_origin" "sr_trusted_origin_preview" {
```

Comment:
What do you think happens here when all 3 of dev, test, and pentest try to create the same `sr_trusted_origin_preview` resource? It might be a no-op or it might be a conflict - I'm not sure. I think there's a better way to do this by having each environment pass in its own list of trusted origins and removing the logic for environment checks.

Reply:
This was a tricky one, as we do have two separate Okta environments: preview and production. I had some difficulty wrapping my head around how to best do this last week, especially since I was seeing a lot of origins for ReportStream that didn't have environment-specific URLs (this is now a non-issue, given that we didn't need to include RS in the first place...my bad!)

Theoretically, in this case, each subsequent environment should be able to read the state change from the previous `terraform apply`, and would just result in a no-op.

Now that we've gotten rid of the ReportStream stuff, and the `localhost` URLs...streamlining this seems really obvious. 😅 I'm happy to make those changes. I'll advise when this is ready for a re-review.

Reply:
One small catch to this, @jdorothy. One of the trusted origin URLs for the production instance of Okta is for `test.simplereport.gov`. The `test` environment itself leverages the Okta Preview environment, which would orphan this specific URL. Since `test` is on the Preview environment, do we want to purge this URL from the list?

Reply:
Good catch! I wonder if this is a relic prior to having a Preview environment. Either way, we shouldn't be allowing production data to flow to non-prod environments so let's purge it.

Reply:
Awesome, thanks for the clarification. Done!